frontend/src/components/common/datepicker/datepicker.tsx

```diff
@@ -16,6 +17,7 @@ type Props = {
   placeholder?: string;
   selectedDate?: Date | string;
   withTime?: boolean;
+  className?: string;
```

Comment:
We don't need to have such customization, datepicker input should take 100% width of it's container, you can limit width by wrapping element in container

Reply:
agreed